bsp4j/src/main/java/ch/epfl/scala/bsp4j/MessageType.java

```diff
@@ -8,7 +8,7 @@ public enum MessageType {
 
     ERROR(1),
     WARNING(2),
-    INFORMATION(3),
+    INFO(3),
```

Comment:
technically should be INFO, though as long as it maps to the right value it shouldn't matter. But we can change it if we still have other breakages

Reply:
we do still have other breakages and I'd prefer not to write a special case for INFO -> INFORMATION transformation